### PR TITLE
changed unrendered emojis

### PR DESCRIPTION
**Pull request by bengtlofgren**

### CODE REVIEW

**Comment by cwgoes** [comment]
@bengtlofgren can you merge this into draft?

**Comment by tzemanovic** [comment]
> @bengtlofgren can you merge this into draft?

docs don't need to go into draft, can be merged directly to main when rdy

**Comment by bengtlofgren** [comment]
can i just get one approval and ill merge :)

**Review by tzemanovic — APPROVED** [review]

**Pull request merged**
